components/input/Search.tsx

```diff
@@ -4,11 +4,13 @@ import Input, { InputProps } from './Input';
 import Icon from '../icon';
 
 export interface SearchProps extends InputProps {
+  inputPrefixCls?: string;
   onSearch?: (value: string) => any;
 }
 
 export default class Search extends React.Component<SearchProps, any> {
   static defaultProps = {
+    inputPrefixCls: 'ant-input',
     prefixCls: 'ant-input-search',
```

Comment:
So the `prefixCls` is unused?

Reply:
No, `prefixCls` is still used to configure the `Icon` and the top-level `className` for `Input`. `inputPrefixCls` is used as a configuration option for `Input` only. This is very similar to how child components within `Slider` and `Pagination` work.